README.md

```diff
@@ -53,6 +53,11 @@ $ pg_migrate -h
 Migrating is supported to the same or newer PostgreSQL version starting from PostgreSQL 9.5 to PostgreSQL 14.
 Migrating to older version is not supported.
 
+By default it searches `pg_dump` under `/usr/`, when using PostgreSQL installs on different directory such as on Mac, use `--pgbin` parameter to define PostgreSQL home directory. e,g,
+```
+--pgbin /Applications/Postgres.app/Contents/Versions
```

Comment:
Is this correct? doesn't the postgres app use something like ```Versions/11.2/bin``` ? This is a genuine question since I can't test that; if that's correct, then that's ok.

Reply:
No it's not correct, thanks for catching this!  I updated the code.